demo-notebooks/guided-demos/0_basic_ray.ipynb

```diff
@@ -45,10 +45,12 @@
    "id": "bc27f84c",
    "metadata": {},
    "source": [
-    "Here, we want to define our cluster by specifying the resources we require for our batch workload. Below, we define our cluster object (which generates a corresponding AppWrapper).\n",
+    "Here, we want to define our cluster by specifying the resources we require for our batch workload. Below, we define our cluster object (which generates a corresponding RayCluster).\n",
     "\n",
     "NOTE: We must specify the `image` which will be used in our RayCluster, we recommend you bring your own image which suits your purposes. \n",
-    "The example here is a community image."
+    "The example here is a community image.\n",
+    "\n",
+    "NOTE: By default the SDK uses Kueue as it's scheduling solution to use MCAD set the `mcad=True` option in `ClusterConfiguration`"
```

Comment:
I think it is better to mention that MCAD component is in Tech Preview along wherever we have mentioned 'mcad=True'

Reply:
I can see the change only in 'guided-demos' notebook. This should be updated in each notebook wherever enabling of mcad is mentioned

Reply:
Oops I missed one thanks

Reply:
I would recommend removing user visible references to MCAD entirely from the demo notebooks.  We'll want to add in an example of using AppWrappers but that's not an alternative to Kueue in the same way MCAD was.

Reply:
@dgrove-oss Sure thing

Reply:
What about the InstaScale demo?
MCAD v1 and InstaScale are still tech preview

Reply:
My understanding is that we are pulling InstaScale as well.  I'd also recommend cutting InstaScale from the demo notebooks.